.npmignore

```diff
@@ -0,0 +1,8 @@
+*
+!/dist/**/*
+!/lib/**/*
+!/es/**/*
+!/docs/**/*
```

Comment:
We need this to be more nuanced because `docs` also has a `node_modules` and lots of non-published stuff.

As a tip you can do:

```
$ npm pack
```

to preview what would be published (it creates a throwaway tarball in the repo).

Reply:
Could probably just ignore docs entirely with regard to npm, yeah?